### ISSUE TITLE
earlystop

### ISSUE THREAD

**Issue by Chenzkun** [ultralytics/yolov5#9013]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

@glenn-jocher，I want to change the map is not rising in the earlystop mechanism condition to verify that the val_loss is not falling , what should i do?, thanks

**Comment by glenn-jocher** [comment]
@Chenzkun EarlyStopper watches to see that fitness stops increasing. You could pass inverse val loss to fitness maybe if you want to use that. Function is here:
https://github.com/ultralytics/yolov5/blob/529aafd737053264cf8676b29c37f5d5300460eb/utils/metrics.py#L15-L19
